### PR TITLE
Standarize Readme and add License

### PR DESCRIPTION
**Pull request by RichardLitt**
See https://github.com/ipfs/community/issues/124

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2015 IPFS
```
Comment (daviddias) [inline LICENSE]: Why not 2016? And I feel this one is @whyrusleeping's repo

**Comment by daviddias** [comment]
Made a comment, LGTM otherwise :)

**Comment by RichardLitt** [comment]
Removed License. @whyrusleeping Can you tell me what License you want on this?

**Comment by RichardLitt** [comment]
@whyrusleeping Thoughts?

**Comment by dignifiedquire** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,10 +1,45 @@
...
+- [Usage](#usage)
+- [Contribute](#contribute)
+  - [Want to hack on IPFS?](#want-to-hack-on-ipfs)
+- [License](#license)
+
+## Install
+
+Right now it depends on changes to [go-ipfs](https://github.com/ipfs/go-ipfs), [js-ipfs-api](https://github.com/ipfs/js-ipfs-api), and feross's [drag-drop](https://github.com/feross/drag-drop) that have not merged. If you want to try this out, there is a bit of pre-setup work you'll need to do:
+
+- Run feat/mfs branch of go-ipfs
```
Comment (dignifiedquire) [inline README.md]: This is now in `master` and in 0.4.3

File: README.md [hunk 3]
```diff
@@ -1,10 +1,45 @@
...
+- Run the daemon in a way that allows different origins
+- Use the js-ipfs-api branch that has mfs support
```
Comment (dignifiedquire) [inline README.md]: This is also part of the released version now

**Comment by RichardLitt** [comment]
Who would be in charge of the license?

**Comment by whyrusleeping** [comment]
MIT

**Comment by RichardLitt** [comment]
Added. Needs final review.

**Comment by RichardLitt** [comment]
This closes #4.

**Pull request merged**